### ISSUE TITLE
private mode

### ISSUE THREAD

**Issue by avenantsoftware** [zsyslog/osx-netflix#1]
playback doesn't work it gives an error about browser being in private mode and in xcode console this error is shown: <<<< MediaValidator >>>> mv_ValidateRFC4281_mp4a: Unrecognized mp4a subelements: a6

Does playback work for you?

**Comment by zsyslog** [comment]
Hello:

Is a silverlight problem. Try:
1. removing Silverlight and reinstall
2. enable Local Storage in silverlight

or apply something like this

WebPreferences\* prefs = [webView preferences];
[prefs _setLocalStorageDatabasePath:@"~/Library/Application Support/OSX-NETFLIX"];
[prefs setLocalStorageEnabled:YES];

cheers

**Comment by avenantsoftware** [comment]
but doesn't netflix since some time now support html5? you can use safari without silverlight for netlix, so why doesn't webview work with html5?

**Comment by zsyslog** [comment]
I don't know if is a Silverlight or LocalSotorage only problem.
Did you tried setting prefs (my 2nd option)?

**Comment by avenantsoftware** [comment]
yeah i tried this, not working, it is related to storage, this is disabled for webview in OS X, I found this hack: http://stackoverflow.com/questions/8198453/local-storage-in-webview-is-not-persistent/18153115#18153115

trying it now...

**Comment by thegitarist** [comment]
any progress? i had no success with the last suggestion. same problem

**Comment by zsyslog** [comment]
No progress at all. I'm thinking about DRM handling. 
![screen shot 2015-09-08 at 10 44 45 am](https://cloud.githubusercontent.com/assets/760671/9736261/7c073774-5616-11e5-8c60-46f1fdf2f5a9.png)
